shop-service/product-service/productsService/index.ts

```diff
@@ -0,0 +1,27 @@
+import PRODUCTS_LIST from '../mockData';
+
+type ProductType = {
+    count: number,
```

Comment:
you may want to use semicolon `;` instead of comma `,` at the end of ProductType's properties

Reply:
fixed